code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -330,6 +330,36 @@
 /mob/living/silicon/ai/cancel_camera()
 	view_core()
 
+/mob/living/silicon/ai/verb/wipe_core()
+	set name = "Wipe Core"
+	set category = "OOC"
+	set desc = "Wipe your core. This is functionally equivalent to cryo, freeing up your job slot."
+
+	// Guard against misclicks, this isn't the sort of thing we want happening accidentally
+	if(alert("WARNING: This will immediately wipe your core and ghost you, removing your character from the round permanently (similar to cryo). Are you entirely sure you want to do this?",
+					"Wipe Core", "No", "No", "Yes") != "Yes")
+		return
+
+	// We warned you.
+	new /obj/structure/AIcore/deactivated(loc)

```

Comment:
I know you'll hate me for it, but we this might cause issues and we're aiming to be issueless.

```
transfer_fingerprints_to()```